src/autosize.js

```diff
@@ -87,11 +89,15 @@ function assign(ta) {
 		const arr = [];
 
 		while (el && el.parentNode && el.parentNode instanceof Element) {
-			if (el.parentNode.scrollTop) {
+			var scrollTop = el.parentNode.scrollTop;
+			if(el.parentNode === document.body){
+				scrollTop = document.documentElement.scrollTop;
+			}
+			if (scrollTop) {
```

Comment:
Is there any need for this check? From what i can see `scrollTop` is always going to have an assignment (either `el.parentNode.scrollTop` or `document.documentElement.scrollTop`)?

Reply:
But doesn't your change make the `if` statement redundant as `scrollTop` will always have an assignment (either `el.parentNode.scrollTop` or `document.documentElement.scrollTop`)

Reply:
> scrollTop will always have an assignment

That was using node.scrollTop. This cannot be used on the body element, therefore the fix where:
```
scrollTop = document.documentElement.scrollTop;

instead of 

scrollTop = body.scrollTop;
```

you get me?

Reply:
Yep i get you, i don't think you get me though 😛

I was trying to assess whether you still needed the `if` statement. On a second glance, the logic in the `if` will not be executed if `el.parentNode.scrollTop` is falsy, and this behaviour is probably still needed as `el.parentNode.scrollTop` can be still be falsy if `el.parentNode !== document.body`.

**TLDR;**

It's fine.

Reply:
but do you still think we need it? I can't see any code path that actually needs that. don't give up on me lol
We'll chat tomorrow, just to make sure.